picocolors.js

```diff
@@ -1,60 +1,60 @@
 let tty = require("tty")
 
 let isColorSupported =
-	!("NO_COLOR" in process.env || process.argv.includes("--no-color")) &&
-	("FORCE_COLOR" in process.env ||
+	!process.env.NO_COLOR &&
+	!process.argv.includes("--no-color") &&
+	(!!process.env.FORCE_COLOR ||
 		process.argv.includes("--color") ||
 		process.platform === "win32" ||
 		(tty.isatty(1) && process.env.TERM !== "dumb") ||
-		"CI" in process.env)
+		!!process.env.CI)
 
-function formatter(open, close, replace = open) {
-	return (input) => {
+let formatter =
+	(open, close, replace = open) =>
+	(input) => {
 		let string = "" + input
 		let index = string.indexOf(close, open.length)
-		return !~index
-			? open + string + close
-			: open + replaceClose(string, close, replace, index) + close
+		return open + (~index ? replaceClose(string, close, replace, index) : string) + close
```

Comment:
This most likely affects benchmarks. I tried to use ternary operator within a concat expression and always saw notable drop in performance (see complex benchmark)

Reply:
True. Updated in a way that is still shorter, but preserves the original logic: 000f178df1c3441952f9d675041526d74b7ab55b.